modules/core/src/test/java/org/apache/ignite/internal/processors/cache/query/continuous/ContinuousQueryMarshallerTest.java

```diff
@@ -104,31 +112,32 @@ private void check(String node1Name, String node2Name) throws Exception {
 
         qry.setRemoteFilterFactory(new DummyEventFilterFactory<>());
 
-        final CountDownLatch latch = new CountDownLatch(15);
+        final CountDownLatch latch = new CountDownLatch(NUM_OF_UPDATES);
 
         qry.setLocalListener(new CacheEntryUpdatedListener<Integer, MarshallerCheckingEntry>() {
             @Override public void onUpdated(
                 final Iterable<CacheEntryEvent<? extends Integer, ? extends MarshallerCheckingEntry>> evts)
                 throws CacheEntryListenerException {
 
-                System.out.println(">> Client 1 events " + evts);
-
                 for (CacheEntryEvent<? extends Integer, ? extends MarshallerCheckingEntry> evt : evts)
                     latch.countDown();
             }
         });
 
         final IgniteCache<Integer, MarshallerCheckingEntry> cache1 = node2.cache(CACHE_NAME);
 
+        int count = 0;
```

Comment:
In accordance with https://cwiki.apache.org/confluence/display/IGNITE/Abbreviation+Rules `cnt` should be used instead of `count`